app/finders/donations_reports_finder.rb

```diff
@@ -0,0 +1,37 @@
+class DonationsReportsFinder
+  def initialize(params)
+    @year = params.dig(:date, :year)
+    @month = params.dig(:date, :month)
+    @cause_id = params[:cause_id]
+    @event_id = params[:event_id]
+    @cause = Cause.find_by(id: @cause_id)
```

Comment:
Looks like `@cause` and `@event` aren't used internally. Let's remove the initialization.